src/broadcast/broadcastManager.ts

```diff
@@ -83,20 +82,20 @@ export class BroadcastManager {
       try {
         await this._connectToDolphin(config.ip, config.port);
       } catch (err) {
-        log.warn(err);
+        this.emit(BroadcastEvent.LOG, err);
```

Comment:
Error is an object. I'm pretty sure subscribers will be expecting a string. So we should make sure we're only return a string.

Reply:
We can probably log `err.message || JSON.stringify(err)`.